tests/cpp/operator/mkldnn_operator_test.cc

```diff
@@ -188,7 +188,7 @@ OpAttrs GetLRNOp() {
   attrs.num_outputs = 2;
   attrs.attrs.dict.insert({"nsize" , "3"});
   attrs.attrs.op->attr_parser(&attrs.attrs);
-  attrs.dispatches.resize(2);
+  attrs.accept_dims.insert(4);  // TODO(alex): (MXNET-845)
```

Comment:
It seems in line 578 you are already testing dims other than 4. Should this TODO be removed?

Reply:
Yes, this TODO is not required now. Removing it.